### PR TITLE
fix(UI): Added new column in exprot spreadsheet in project tab

### PR DESCRIPTION
**Pull request by nikkuma7**
Signed-off-by: Nikesh kumar <kumar.nikesh@simens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
Added a new column and fill the data for respective colunm for export spreadsheet in project tab.
> * Which issue is this pull request belonging to and how is it solving it? #1744 
> * Did you add or update any new dependencies that are required for your change?

Issue: 
Few column was not available when we export spreadsheet in project tab.

project responsible
project owner
security responsibles
flag whether security monitoring is enabled?
release: external IDs

### How To Test?
1: login into sw360
2: navigate to project tab and export spreadsheet.

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ProjectSummary.java [hunk 1]
```diff
@@ -53,6 +53,10 @@ protected static void setDefaultFields(Project document, Project copy) {
         copyField(document, copy, _Fields.DESCRIPTION);
         copyField(document, copy, _Fields.VERSION);
         copyField(document, copy, _Fields.CLEARING_TEAM);
+        copyField(document, copy, _Fields.PROJECT_RESPONSIBLE);
+        copyField(document, copy, _Fields.PROJECT_OWNER);
+        copyField(document, copy, _Fields.SECURITY_RESPONSIBLES);
+        copyField(document, copy, _Fields.ENABLE_VULNERABILITIES_DISPLAY);
```
Comment (akapti) [inline backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ProjectSummary.java:59]: No changes are needed in this file (ProjectSummary). Kindly revert this change.

File: backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ReleaseSummary.java [hunk 2]
```diff
@@ -109,6 +109,7 @@ private void setShortSummaryFields(Release document, Release copy) {
         copyField(document, copy, _Fields.RELEASE_DATE);
         copyField(document, copy, _Fields.SOURCE_CODE_DOWNLOADURL);
         copyField(document, copy, _Fields.BINARY_DOWNLOADURL);
+        copyField(document, copy, _Fields.EXTERNAL_IDS);
```
Comment (akapti) [inline backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ReleaseSummary.java:112]: No changes are needed in this file (ReleaseSummary). Kindly revert this change.

**Review by akapti — CHANGES_REQUESTED** [review]
Tested the changes, and it's working fine.
Please incorporated the review comments.

File: backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ReleaseSummary.java (hunk 2 above)
Comment (nikkuma7) [reply backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ReleaseSummary.java]: done.

File: backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ProjectSummary.java (hunk 1 above)
Comment (nikkuma7) [reply backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ProjectSummary.java]: done.

File: libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ProjectExporter.java [hunk 3]
```diff
@@ -62,6 +62,8 @@ public class ProjectExporter extends ExcelExporter<Project, ProjectHelper> {
         nameToDisplayName.put(OWNER_GROUP.getFieldName(), "owner group");
         nameToDisplayName.put(OWNER_COUNTRY.getFieldName(), "owner country");
         nameToDisplayName.put(VENDOR_ID.getFieldName(), "vendor id");
+        nameToDisplayName.put(SECURITY_RESPONSIBLES.getFieldName(), "security responsibles");
+        nameToDisplayName.put(ENABLE_VULNERABILITIES_DISPLAY.getFieldName(), "enableVulnerabilitiesDisplay");
```
[the remark this answers is not in the document]
Comment (nikkuma7) [reply libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ProjectExporter.java:65]: done.

**Review by akapti — APPROVED** [review]
looks good to me.
tested the changes and project excel sheet contains the required information.
